src/stdune/once.mli

```diff
@@ -0,0 +1,27 @@
+(** Cached lazy evaluation *)
```

Comment:
This module is now unused, so we can remove it from this PR.

Reply:
removed it.